src/main/java/de/zalando/aruha/nakadi/annotations/DB.java

```diff
@@ -0,0 +1,17 @@
+package de.zalando.aruha.nakadi.annotations;
+
+import org.springframework.beans.factory.annotation.Qualifier;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target({ElementType.FIELD,
+        ElementType.METHOD,
+        ElementType.TYPE,
+        ElementType.PARAMETER})
+@Retention(RetentionPolicy.RUNTIME)
+@Qualifier
+public @interface DB {
```

Comment:
I specify to inject particular EventTypeRepository (EventTypeDBRepository) in CachingEventTypeRepository